src/main/java/com/wastedbankspace/WastedBankSpacePlugin.java

```diff
@@ -250,10 +254,16 @@ private void updateWastedBankSpace()
 
 	public List<StorableItem>  getEnabledItemLists()
 	{
+		NonFlaggedItemList = Text.fromCSV(config.getNonFlaggedItems());
+
 		List<StorableItem> ret = new ArrayList<>();
 		for (StorageLocationEnabler sle:
 				storageLocationEnablers) {
-			ret.addAll(sle.GetStorableItems());
+			for(StorableItem item: sle.GetStorableItems()){
+				if (!NonFlaggedItemList.contains(item.name)) {
```

Comment:
If we keep storing the Flagged items as plain text, than its faster to do name checking instead of, first mapping the flagged item names to an item to get the ID and than do the if check.

Reply:
I do not agree with that statement. For now it should be fine, but when I merge I will branch and play around with the alternative.